api/advisory/transform_to_epsg_3005.py

```diff
@@ -0,0 +1,101 @@
+import sys
```

Comment:
This feels like a generalize geospatial utility. What do you think about throwing it into api/app/utils ?